demo/agent.cc

```diff
@@ -83,7 +83,7 @@ int main(int argc, char* argv[]) {
   }
 
   auto handler = use_queue
-      ? std::make_unique<QueuingHandler>(delay, save_print_data_path)
+      ? std::make_unique<QueuingHandler>(8u, delay, save_print_data_path)
```

Comment:
Won't this remove the flag support for custom delay?

Reply:
How come? We can still set the same custom delay for all the threads right? Or you mean setting different delays

Reply:
I don't think so.  Could add a new flag to support the number of threads to use in the pool.

Reply:
I added new flag for num of threads since Nancy also asked about that.